crates/interpreter/src/instructions/system.rs

```diff
@@ -21,12 +21,12 @@ pub fn keccak256<H: Host>(interpreter: &mut Interpreter, _host: &mut H) {
 
 pub fn address<H: Host>(interpreter: &mut Interpreter, _host: &mut H) {
     gas!(interpreter, gas::BASE);
-    push_b256!(interpreter, B256::from(interpreter.contract.address));
+    push_b256!(interpreter, interpreter.contract.address.into_word());
 }
 
 pub fn caller<H: Host>(interpreter: &mut Interpreter, _host: &mut H) {
     gas!(interpreter, gas::BASE);
-    push_b256!(interpreter, B256::from(interpreter.contract.caller));
+    push_b256!(interpreter, interpreter.contract.caller.into_word());
```

Comment:
I get the reference as in processor word (native type), but `into_word` feels strange to me. Bikeshedding here, nothing to be changed in the essence, maybe `into_evm_word` would be a more obvious name, but it does increase the fn name size.